src/bin/vip-dev-env-import-sql.js

```diff
@@ -64,6 +65,12 @@ command( {
 		try {
 			const resolvedPath = await resolveImportPath( slug, fileName, searchReplace, inPlace );
 
+			// Remove the `USE` and `CREATE DATABASE` statements from the SQL file.
+			ChildProcess.execFileSync(
+				'sed',
```

Comment:
Would this work or would you have to provide the path to the actual executable file? `

Reply:
`sed` is a standard linux command, so this should work out of the box.

Reply:
Ah.... this will be executed on user's OS. And `sed` is not available on windows. :grimacing:

Reply:
yup 😄

Reply:
On another thought, we don't support vip-cli on windows natively. One has to install windows subsystem for linux to be able to vip-cli on windows: https://docs.wpvip.com/technical-references/vip-cli/installing-vip-cli/

And `sed` is available on WSL.